onnxruntime/core/graph/graph.cc

```diff
@@ -926,6 +928,47 @@ ADD_ATTR_IMPLS(TypeProto)
 #undef ADD_ATTR_LIST_IMPL
 #undef ADD_ATTR_IMPLS
 
+#ifndef NDEBUG
+void Node::AddAttribute(std::string attr_name, int64_t value) {
```

Comment:
Is this a temporary change? If not, I don't think we should be doing this in graph.cc. Doesn't the ONNX checker do this validation?

Reply:
Yes, i added it to detect issues due to optimizers,i plan to remove it because testing everything would take too much time here.

Reply:
But that way it is faster to find the line adding the wrong node.

Reply:
removed